portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +27,48 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  
+  private static final Gson gson = new Gson();
+  private ArrayList<String> comments;
+
+  @Override
+  public void init() {
+    comments = new ArrayList<>();
```

Comment:
is it supposed to be the case that all comments are removed if your web server restarts? or are you supposed to write to the datastore and read from the datastore and not keep this field here?

with the current design, these comments are only kept in memory. also HttpServlet is by default multi threaded and an ArrayList does not allow concurrent access

Reply:
for this particular PR, comments were only being stored in this ArrayList. The datastore is implemented in the next PR.